styles/components/card.css

```diff
@@ -0,0 +1,43 @@
+.card {
+  background-color: yellow;
+  padding: 0 16px 0;
+  display: grid;
+  flex-direction: column;
+  border-radius: 8px;
+  height: 320px;
+  position: relative;
+  align-items: center;
+}
+
+.card__button {
+  justify-content: center;
```

Comment:
I think this has to be in the container element. thats how i understand the grid cheat sheet :) if its working there, i guess ist fine too.

Reply:
yeah thank you, its a completely useless piece of code :D